install.sh

```diff
@@ -390,6 +391,20 @@ End of functions from https://github.com/client9/shlib
 ------------------------------------------------------------------------
 EOF
 
+cosign_verify() {
+  if ! is_command cosign; then
+    log_info "cosign is not available, skipping verify"
+    return 0
+  fi
+  checksums=$1
+  http_download "${tmpdir}/${CHECKSUM_COSIGN_BUNDLE}" "${CHECKSUM_COSIGN_BUNDLE_URL}"
```

Comment:
This will fail for releases that don't have signed checksums. I suppose we should address it somehow, but I'm not sure what would be the best way. Some thoughts:
- Hardcode some version comparison (chore in shell script code), don't try it for versions known to not have it?
- Skip verify if downloading the cosign bundle fails due to HTTP 404? Not sure if there's a good way to implement that for wget.

Reply:
An example of this problem can be seen in the CI failure of this PR.